scripts/set-githooks.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+git config --local core.hooksPath .githooks/
```

Comment:
I think a more common setup would be https://github.com/lint-staged/lint-staged?tab=readme-ov-file#installation-and-setup